dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/api/Aggregate.kt

```diff
@@ -89,4 +89,14 @@ interface Aggregate<ID : Any> {
      * Returns the body's return element.
      */
     fun <R> alignedOn(pivot: Any?, body: () -> R): R
+
+    /**
+     * Pushes the pivot in the alignment stack.
+     */
+    fun alignRaw(pivot: Any?)
```

Comment:
Is it possible to hide the function but still call it with plugin magic?

Reply:
I've tried, but no way...